src/core/common/inviwocore.cpp

```diff
@@ -332,6 +332,9 @@ InviwoCore::InviwoCore(InviwoApplication* app)
     using Vec3s = std::tuple<vec3, dvec3, ivec3, size3_t>;
     using Vec4s = std::tuple<vec4, dvec4, ivec4, size4_t>;
 
+    registerPropertyConverter(std::move(std::make_unique<FileToStringConverter>()));
```

Comment:
no need to cast to R-value with std::move since the return value of make_unique already is a R-value by definition.